client/pages/api/create-session.js

```diff
@@ -1,25 +1,22 @@
-const createSessionBackend = `http://127.0.0.1:5000/session`;
+const createSessionBackend = `${process.env.API_HOST}/session`;
```

Comment:
When I set this in my root `.env` file it seems like it might not be picked up. Is it possible to configure next to load .env vars from the outer dir? Ideally the user could just use the same .env file to specify everything they need for both the server and client when running this locally.

Reply:
Next.js automatically reads `.env` vars from the same directory, so making it read from the top-level `.env` var adds some potentially unnecessary complexity to the demo: we'd have to add a Next.js config file, read the `.env` file from the parent directory, and then pass the env variables to the Next.js config. I'd prefer sticking with this separation for now, as technically `API_HOST` shouldn't be of interest for the server and all the tokens shouldn't be of interest for the client app.

Reply:
So to get the client running the user would need to create another .env file in the `client` subdirectory? Let's be sure to include an example file and mention this in the README instructions as well if so.

Reply:
I made the `API_HOST` env var optional and fallback to `http://127.0.0.1:5000`, as proposed!